src/tarantool.client/Box.cs

```diff
@@ -35,10 +41,35 @@ public Box(ConnectionOptions options)
 
             _responseReader.BeginReading();
 
-            await TryLogin(greetings);
+            await LoginIfNotGuest(greetings);
+        }
+
+        public void Dispose()
+        {
+            _physicalConnection.Dispose();
+        }
+
+        public Schema GetSchema()
+        {
+            return new Schema(_logicalConnection);
+        }
+
+        public async Task<DataResponse<TResponse[]>>  Call<TTuple, TResponse>(string functionName, TTuple request)
+            where TTuple : ITuple
+            where TResponse : ITuple
+        {
+            var callRequest = new CallRequest<TTuple>(functionName, request);
+            return await _logicalConnection.SendRequest<CallRequest<TTuple>, TResponse>(callRequest);
+        }
+
+        public async Task<DataResponse<TResponse[]>> Eval<TTuple, TResponse>(string expression, TTuple request)
+           where TTuple : ITuple
+        {
+            var evalRequest = new EvalRequest<TTuple>(expression, request);
+            return await _logicalConnection.SendRequest<EvalRequest<TTuple>, TResponse>(evalRequest);
         }
 
-        private async Task TryLogin(GreetingsResponse greetings)
+        private async Task LoginIfNotGuest(GreetingsResponse greetings)
         {
```

Comment:
We need to provide more meaningful message. A rule of thumb:
- What's happening
- How to fix